webview-ui/src/components/chat/Announcement.tsx

```diff
@@ -120,15 +67,13 @@ const Announcement = ({ version, hideAnnouncement }: AnnouncementProps) => {
 				}}
 			/>
 			<p style={{ margin: "0" }}>
-				Join our{" "}
-				<VSCodeLink style={{ display: "inline" }} href="https://discord.gg/cline">
-					discord
-				</VSCodeLink>{" "}
-				or{" "}
-				<VSCodeLink style={{ display: "inline" }} href="https://www.reddit.com/r/cline/">
-					r/cline
-				</VSCodeLink>
-				for more updates!
+				<Trans
+					i18nKey="announcement.joinOurCommunities"
+					components={{
+						DiscordLink: <VSCodeLink href="https://discord.gg/cline">Discord</VSCodeLink>,
+						RedditLink: <VSCodeLink href="https://www.reddit.com/r/cline/">Reddit</VSCodeLink>,
```

Comment:
The label is defined in the locales file, so it will probably be defined as follows.

```suggestion
						DiscordLink: <VSCodeLink href="https://discord.gg/cline" />,
						RedditLink: <VSCodeLink href="https://www.reddit.com/r/cline/" />,
```